pkg/build/nodeimage/internal/container/docker/archive.go

```diff
@@ -62,17 +62,25 @@ func GetArchiveTags(path string) ([]string, error) {
 	if err != nil {
 		return nil, err
 	}
-	// parse
-	repoTags, err := parseRepositories(b)
-	if err != nil {
-		return nil, err
-	}
-	// convert to tags in the docker CLI sense
 	res := []string{}
-	for repo, tags := range repoTags {
-		for tag := range tags {
-			res = append(res, fmt.Sprintf("%s:%s", repo, tag))
+	// parse
+	if hdr.Name == "repositories" {
+		repoTags, err := parseRepositories(b)
+		if err != nil {
+			return nil, err
+		}
+		// convert to tags in the docker CLI sense
+		for repo, tags := range repoTags {
+			for tag := range tags {
+				res = append(res, fmt.Sprintf("%s:%s", repo, tag))
+			}
 		}
+	} else if hdr.Name == "manifest.json" {
+		manifest, err := parseDockerV1Manifest(b)
+		if err != nil {
+			return nil, err
+		}
+		res = append(res, manifest[0].RepoTags...)
```

Comment:
I think that this failed before if was not able to parse it because at some points the marshalling will not work, and now it just returns an empty `[]string{},nil` ,
Shouldn't we keep failing if neither `manifest.json`or `repositories`is found?

maybe you can use a 
```go
switch hdr.Name:
 case manifest.json:
  // do something
 case repositories:
  // do something
 default:
   return nil, fmt.Errorf("unable to find tags, not compatible with specs ... 
```